tests/conftest.py

```diff
@@ -1,9 +1,172 @@
 """Fixtures common across tests."""
+import pickle
+from typing import Callable, List
 
+import gym
 import pytest
 import torch
+from stable_baselines3 import PPO
+from stable_baselines3.common.policies import BasePolicy
+from stable_baselines3.common.vec_env import DummyVecEnv, VecNormalize
 
+from imitation.data import rollout
+from imitation.data.types import TrajectoryWithRew
+from imitation.data.wrappers import RolloutInfoWrapper
+from imitation.policies.base import FeedForward32Policy, NormalizeFeaturesExtractor
 from imitation.util import logger
+from imitation.util.networks import RunningNorm
+
+CARTPOLE_ENV_NAME = "CartPole-v1"
+
+
+def load_or_train_ppo(
+    cache_path: str,
+    training_function: Callable[[gym.Env], PPO],
+    venv,
+) -> PPO:
+    try:
+        return PPO.load(cache_path, venv)
+    except OSError:
+        pass
+    except AssertionError:
```

Comment:
When do we expect to see this? Internal checks of SB3 that could fail across versions?

Reply:
Also, it might be nice to log the traceback/exception information so that we can tell why policy was regenerated, even though it's no longer a hard-fail.

Reply:
Yes, I had AssertionErrors when loading outdated model files. This is why I added it there. Initially I just tried to catch `Exception` but PEP8 does not approve of that.
I now added a warning when the policy is retrained, rerolled and a comment to explain the AssertionErrors.